### ISSUE TITLE
[BUG] The BOT is restarted several times causing it to consume memory

### ISSUE THREAD

**Issue by XtoMHA96** [kyb3r/modmail#2851]
**Describe the bug**
In these last weeks the BOT has been restarting on several occasions, causing the BOT to consume almost all the memory of the VPS. I have had to stop it on several occasions and put it back to work to clean the memory consumed.

**Bot Info**
Bot version (check with `@modmail about`): **v3.5.0**
Host method (Heroku, self-host, etc): **self-host in a VPS**

**To Reproduce**
1. Who can reproduce (ex. anyone, owners)? **The persons authorized to be able to run the BOT**
2. Where can it be reproduced (ex. in thread channels, recipient DM's)? **In the terminal**
3. Done what to cause the error? **Unknown**
4. Any recently made changes to your bot? **No**

**Comment by XtoMHA96** [comment]
**Error Logs**
```
root@vmi354253:~/DiscordBots/modmail-rm# cd ../modmail-willy/
root@vmi354253:~/DiscordBots/modmail-willy# pipenv run bot
Loading .env environment variables…
09/05/20 22:22:01 __main__[117] - INFO: -------------------------
09/05/20 22:22:01 __main__[125] - INFO: Logging level: INFO
09/05/20 22:22:01 __main__[127] - INFO: Log file: /root/DiscordBots/modmail-willy/temp/NjkyNzYyNTIyODMwMjQxOTAy.log
09/05/20 22:22:01 __main__[91] - INFO: -------------------------
09/05/20 22:22:01 __main__[92] - INFO: ┌┬┐┌─┐┌┬┐┌┬┐┌─┐┬┬
09/05/20 22:22:01 __main__[93] - INFO: ││││ │ │││││├─┤││
09/05/20 22:22:01 __main__[94] - INFO: ┴ ┴└─┘─┴┘┴ ┴┴ ┴┴┴─┘
09/05/20 22:22:01 __main__[95] - INFO: v3.5.0
09/05/20 22:22:01 __main__[96] - INFO: Authors: kyb3r, fourjr, Taaku18
09/05/20 22:22:01 __main__[97] - INFO: -------------------------
09/05/20 22:22:29 cogs.utility[606] - INFO: -------------------------
09/05/20 22:22:29 cogs.utility[615] - INFO: Activity set to: Watching MDs. Envíame un mensaje para reportes o problemas en el servidor..
09/05/20 22:22:29 cogs.utility[620] - INFO: Status set to: online.
09/05/20 22:22:29 __main__[391] - INFO: -------------------------
09/05/20 22:22:29 __main__[393] - INFO: Logged in as: Modmail#1523
09/05/20 22:22:29 __main__[394] - INFO: Bot ID: 692762522830241902
09/05/20 22:22:29 __main__[399] - INFO: Owners: Paul, XtoMHA96
09/05/20 22:22:29 __main__[400] - INFO: Prefix: .
09/05/20 22:22:29 __main__[401] - INFO: Guild Name: Willyrex! 🔥 🦖
09/05/20 22:22:29 __main__[402] - INFO: Guild ID: 418586876899885056
09/05/20 22:22:29 __main__[405] - INFO: -------------------------
09/05/20 22:22:29 __main__[411] - INFO: There are 5 thread(s) pending to be closed.
09/05/20 22:22:29 __main__[412] - INFO: -------------------------
09/05/20 22:37:07 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/05/20 22:38:02 __main__[1159] - WARNING: CommandNotFound: Command "......." is not found
09/05/20 22:52:29 cogs.utility[625] - INFO: Starting presence loop.
09/05/20 23:13:40 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 00:01:02 __main__[391] - INFO: -------------------------
09/06/20 00:01:02 __main__[393] - INFO: Logged in as: Modmail#1523
09/06/20 00:01:02 __main__[394] - INFO: Bot ID: 692762522830241902
09/06/20 00:01:02 __main__[399] - INFO: Owners: Paul, XtoMHA96
09/06/20 00:01:02 __main__[400] - INFO: Prefix: .
09/06/20 00:01:02 __main__[401] - INFO: Guild Name: Willyrex! 🔥 🦖
09/06/20 00:01:02 __main__[402] - INFO: Guild ID: 418586876899885056
09/06/20 00:01:02 __main__[405] - INFO: -------------------------
09/06/20 00:01:02 __main__[411] - INFO: There are 5 thread(s) pending to be closed.
09/06/20 00:01:02 __main__[412] - INFO: -------------------------
09/06/20 00:59:31 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 01:12:32 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 03:03:14 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 03:11:35 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 03:27:15 __main__[1159] - WARNING: CommandNotFound: Command "." is not found
09/06/20 03:35:51 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 03:37:38 __main__[1159] - WARNING: CommandNotFound: Command "__________________________:" is not found
09/06/20 04:06:10 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 04:20:43 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 04:33:08 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 04:38:01 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 04:48:13 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 05:25:28 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 05:32:55 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 05:34:16 __main__[1159] - WARNING: CommandNotFound: Command "__." is not found
09/06/20 05:44:24 __main__[1159] - WARNING: CommandNotFound: Command "_.xd" is not found
09/06/20 05:48:43 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 06:19:37 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 06:32:12 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 06:50:30 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 07:05:50 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 07:40:16 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 07:45:19 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 07:51:24 __main__[1159] - WARNING: CommandNotFound: Command "y." is not found
09/06/20 07:53:17 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 08:07:17 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 09:14:29 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 09:20:50 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 09:29:46 __main__[391] - INFO: -------------------------
09/06/20 09:29:46 __main__[393] - INFO: Logged in as: Modmail#1523
09/06/20 09:29:46 __main__[394] - INFO: Bot ID: 692762522830241902
09/06/20 09:29:46 __main__[399] - INFO: Owners: Paul, XtoMHA96
09/06/20 09:29:46 __main__[400] - INFO: Prefix: .
09/06/20 09:29:46 __main__[401] - INFO: Guild Name: Willyrex! 🔥 🦖
09/06/20 09:29:46 __main__[402] - INFO: Guild ID: 418586876899885056
09/06/20 09:29:46 __main__[405] - INFO: -------------------------
09/06/20 09:29:46 __main__[411] - INFO: There are 8 thread(s) pending to be closed.
09/06/20 09:29:46 __main__[412] - INFO: -------------------------
09/06/20 09:37:42 __main__[1159] - WARNING: CommandNotFound: Command "_.<:peepo_cringe:693526105293848618>" is not found
09/06/20 10:05:49 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 10:22:10 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 10:34:13 __main__[1159] - WARNING: CommandNotFound: Command "__." is not found
09/06/20 10:57:55 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 10:59:31 __main__[1159] - WARNING: CommandNotFound: Command "_?" is not found
09/06/20 10:59:46 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 11:03:07 __main__[1159] - WARNING: CommandNotFound: Command "__." is not found
09/06/20 11:09:17 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 11:12:49 __main__[1159] - WARNING: CommandNotFound: Command "..." is not found
09/06/20 11:13:01 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 11:16:19 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 11:18:00 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 11:28:09 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 11:31:35 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 11:32:45 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 11:40:01 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 11:40:07 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 11:50:27 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 12:00:41 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 12:13:52 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 12:14:56 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 12:17:58 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 12:18:14 __main__[1159] - WARNING: CommandNotFound: Command "-.xd" is not found
09/06/20 12:36:59 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 13:17:58 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 13:20:43 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 13:41:31 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 13:46:20 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 13:50:48 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 14:27:53 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 15:20:51 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 16:18:52 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 16:19:50 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 16:29:51 __main__[1159] - WARNING: CommandNotFound: Command "." is not found
09/06/20 17:20:05 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 17:20:45 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 18:02:24 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 19:02:31 __main__[1159] - WARNING: CommandNotFound: Command "v." is not found
09/06/20 19:24:13 __main__[1159] - WARNING: CommandNotFound: Command "v." is not found
09/06/20 19:44:57 core.thread[635] - ERROR: Message delivery failed:
Traceback (most recent call last):
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 632, in reply
    message, destination=self.recipient, from_mod=True, anonymous=anonymous
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 848, in send
    msg = await destination.send(mentions, embed=embed)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/abc.py", line 870, in send
    data = await state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/http.py", line 221, in request
    raise Forbidden(r, data)
discord.errors.Forbidden: 403 Forbidden (error code: 50007): Cannot send messages to this user
09/06/20 19:45:00 core.thread[635] - ERROR: Message delivery failed:
Traceback (most recent call last):
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 632, in reply
    message, destination=self.recipient, from_mod=True, anonymous=anonymous
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 848, in send
    msg = await destination.send(mentions, embed=embed)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/abc.py", line 870, in send
    data = await state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/http.py", line 221, in request
    raise Forbidden(r, data)
discord.errors.Forbidden: 403 Forbidden (error code: 50007): Cannot send messages to this user
09/06/20 19:45:11 __main__[1185] - ERROR: Unexpected exception:
Traceback (most recent call last):
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/core.py", line 83, in wrapped
    ret = await coro(*args, **kwargs)
  File "/root/DiscordBots/modmail-willy/cogs/modmail.py", line 386, in close
    await thread.close(closer=ctx.author, after=close_after, message=message, silent=silent)
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 295, in close
    await self._close(closer, silent, delete_channel, message)
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 400, in _close
    await asyncio.gather(*tasks)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/abc.py", line 870, in send
    data = await state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/http.py", line 221, in request
    raise Forbidden(r, data)
discord.errors.Forbidden: 403 Forbidden (error code: 50007): Cannot send messages to this user

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/bot.py", line 892, in invoke
    await ctx.command.invoke(ctx)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/core.py", line 797, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/core.py", line 92, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: Forbidden: 403 Forbidden (error code: 50007): Cannot send messages to this user
09/06/20 19:45:32 __main__[1185] - ERROR: Unexpected exception:
Traceback (most recent call last):
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/core.py", line 83, in wrapped
    ret = await coro(*args, **kwargs)
  File "/root/DiscordBots/modmail-willy/cogs/modmail.py", line 386, in close
    await thread.close(closer=ctx.author, after=close_after, message=message, silent=silent)
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 295, in close
    await self._close(closer, silent, delete_channel, message)
  File "/root/DiscordBots/modmail-willy/core/thread.py", line 400, in _close
    await asyncio.gather(*tasks)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/abc.py", line 870, in send
    data = await state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/http.py", line 221, in request
    raise Forbidden(r, data)
discord.errors.Forbidden: 403 Forbidden (error code: 50007): Cannot send messages to this user

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/bot.py", line 892, in invoke
    await ctx.command.invoke(ctx)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/core.py", line 797, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "/root/.local/share/virtualenvs/modmail-willy-BVTNA0We/lib/python3.7/site-packages/discord/ext/commands/core.py", line 92, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: Forbidden: 403 Forbidden (error code: 50007): Cannot send messages to this user
09/06/20 20:51:11 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 20:51:57 __main__[1159] - WARNING: CommandNotFound: Command "v." is not found
09/06/20 20:54:28 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 20:54:33 __main__[1159] - WARNING: CommandNotFound: Command "..." is not found
09/06/20 21:17:38 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 21:21:52 __main__[1159] - WARNING: CommandNotFound: Command "..................bendo" is not found
09/06/20 21:24:18 __main__[1159] - WARNING: CommandNotFound: Command ".............." is not found
09/06/20 22:18:37 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 22:29:36 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 22:30:30 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 22:56:03 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 23:01:15 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:01:19 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:01:23 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:08:28 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:09:12 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/06/20 23:09:58 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:10:21 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:10:46 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:11:59 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:13:08 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:13:11 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:15:41 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:20:07 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/06/20 23:33:52 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/06/20 23:45:13 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 00:38:13 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 00:49:02 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 01:34:13 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/07/20 01:34:46 __main__[1159] - WARNING: CommandNotFound: Command "/ban" is not found
09/07/20 01:35:02 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 02:15:19 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 02:18:39 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 03:12:18 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 04:40:28 __main__[1159] - WARNING: CommandNotFound: Command "." is not found
09/07/20 04:40:57 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 04:41:30 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 04:58:52 __main__[1159] - WARNING: CommandNotFound: Command "work" is not found
09/07/20 05:01:52 __main__[1159] - WARNING: CommandNotFound: Command "__." is not found
09/07/20 05:19:28 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 05:49:29 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 05:49:42 __main__[391] - INFO: -------------------------
09/07/20 05:49:42 __main__[393] - INFO: Logged in as: Modmail#1523
09/07/20 05:49:42 __main__[394] - INFO: Bot ID: 692762522830241902
09/07/20 05:49:42 __main__[399] - INFO: Owners: Paul, XtoMHA96
09/07/20 05:49:42 __main__[400] - INFO: Prefix: .
09/07/20 05:49:42 __main__[401] - INFO: Guild Name: Willyrex! 🔥 🦖
09/07/20 05:49:42 __main__[402] - INFO: Guild ID: 418586876899885056
09/07/20 05:49:42 __main__[405] - INFO: -------------------------
09/07/20 05:49:42 __main__[411] - INFO: There are 5 thread(s) pending to be closed.
09/07/20 05:49:42 __main__[412] - INFO: -------------------------
09/07/20 05:49:51 __main__[1159] - WARNING: CommandNotFound: Command "_.xd" is not found
09/07/20 05:52:33 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 05:58:16 __main__[1159] - WARNING: CommandNotFound: Command "__." is not found
09/07/20 06:08:45 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 06:18:45 __main__[1159] - WARNING: CommandNotFound: Command ".." is not found
09/07/20 06:32:54 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 06:37:51 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 07:28:11 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 07:47:49 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 09:26:05 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 09:38:28 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 09:39:48 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 09:43:38 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 09:44:08 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 09:47:27 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 09:48:09 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
09/07/20 09:50:59 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 09:53:34 __main__[1159] - WARNING: CommandNotFound: Command "_." is not found
09/07/20 10:05:21 __main__[1159] - WARNING: CommandNotFound: Command "work" is not found
09/07/20 10:06:18 __main__[1159] - WARNING: CommandNotFound: Command "-." is not found
^C09/07/20 10:16:14 __main__[178] - ERROR:  - Shutting down bot - 

```

**Comment by matrix2113** [comment]
Dupe of #2801

An issue was resolved, hopefully if you are running the latest version of discord.py. Please note that there are many factors to this issue and it is continued to being looked into.

**Comment by PaeP3nguin** [comment]
I was looking through the changelog for discord.py and I think this was fixed in version 1.4.0: https://discordpy.readthedocs.io/en/latest/whats_new.html#id8. The version this repo uses right now is 1.3.4, so you may need to upgrade manually to see the changes.

**Comment by fourjr** [comment]
Should be fixed in v3.6.2
